### ISSUE TITLE
RC Test: Molecules and Light 1.4.0-rc.1

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#84]
@emily-phet @ariel-phet @phet-steele, Molecules and Light 1.4.0-rc.1 is ready for RC testing. This version of the sim includes keyboard navigation. Instructions for how to interact with each sim component with a keyboard can be found in the "Keyboard Help Dialog".

**[Link to sim](https://www.colorado.edu/physics/phet/dev/html/molecules-and-light/1.4.0-rc.1/molecules-and-light_en.html)**

**[Link to iFrame](https://www.colorado.edu/physics/phet/dev/html/molecules-and-light/1.4.0-rc.1/molecules-and-light_en-iframe.html)**

**[Test Matrix](https://docs.google.com/spreadsheets/d/1OUzxwzBS3Bpws1zu5sM5st-b7dNzN-f08EHxvIW4qoA/edit#gid=1361634836)**

**Please also verify**
- [x] stringTest=double (all strings doubled)
- [x] stringTest=long (exceptionally long strings)
- [x] stringTest=X (short strings)
- [x] stringTest=rtl (right-to-left)
- [x] stringTest=xss (test passes if sim does not redirect, OK if sim crashes or fails to fully start)
- [x] showPointerAreas (touchArea=red, mouseArea=blue)
- [x] Full screen test
- [x] Screenshot test
- [x] Does AR need to be notified for any LOL updates?

If any new issues are found, please note them in https://github.com/phetsims/molecules-and-light/issues and reference this issue.  

@ariel-phet can you please prioritize?

**Comment by lmulhall-phet** [comment]
Is it normal for the molecule strings to be unchanged under stringTest=xss?

![screen shot 2018-01-26 at 3 07 38 pm](https://user-images.githubusercontent.com/31285590/35462773-b544b276-02aa-11e8-97b9-59d494b551c6.png)

**Comment by lmulhall-phet** [comment]
Other than the above comment, looks good.

**Comment by phet-steele** [comment]
> Is it normal for the molecule strings to be unchanged under stringTest=xss?

@lmulhall-phet yes this is normal. Those string are written in RichText, which does not change with `?stringTest=xss`

**Comment by phet-steele** [comment]
El fin.

**Comment by jessegreenberg** [comment]
Thank you @phet-steele and team! https://github.com/phetsims/molecules-and-light/issues/175 was the only issue and it has been resolved. Closing we will have a spot check RC out soon.
